src/compute/grid/track_sizing.rs

```diff
@@ -687,7 +707,7 @@ fn expand_flexible_tracks(
         AvailableSpace::Definite(available_space) => {
             let used_space: f32 = axis_tracks.iter().map(|track| track.base_size).sum();
             let free_space = available_space - used_space;
-            if free_space == 0.0 {
+            if free_space <= 0.0 {
```

Comment:
It can yeah. This would be the case if the containing node has a fixed size (set with the width/height property), and the contents of the the container had minimum sizes that sum to total greater than the size of the container (so the contents overflow). I hadn't considered this case either, but it came up when debugging a test, hence the fix!